doc/guide/install.rst

```diff
@@ -8,25 +8,28 @@ The following sections will guide you through the installation of CLIMADA and it
 
     CLIMADA has a complicated set of dependencies that cannot be installed with ``pip`` alone.
     Please follow the installation instructions carefully!
-    We recommend to use `Anaconda`_ for creating a suitable software environment to execute CLIMADA.
+    We recommend to use `Conda`_ for creating a suitable software environment to execute CLIMADA.
 
-All following instructions should work on any operating system (OS) that is supported by `Anaconda`_, including in particular: **Windows**, **macOS**, and **Linux**.
+All following instructions should work on any operating system (OS) that is supported by `Conda`_, including in particular: **Windows**, **macOS**, and **Linux**.
 
-.. note:: When mentioning the terms "terminal" or "command line" in the following, we are referring to the "Terminal" apps on macOS or Linux and the "Anaconda Prompt" on Windows.
+.. hint:: If you need help with the vocabulary used on this page, refer to the :ref:`Glossary <install-glossary>`.
 
 -------------
 Prerequisites
 -------------
 
 * Make sure you are using the **latest version** of your OS. Install any outstanding **updates**.
 * Free up at least 10 GB of **free storage space** on your machine.
-  Anaconda and the CLIMADA dependencies will require around 5 GB of free space, and you will need at least that much additional space for storing the input and output data of CLIMADA.
+  Conda and the CLIMADA dependencies will require around 5 GB of free space, and you will need at least that much additional space for storing the input and output data of CLIMADA.
 * Ensure a **stable internet connection** for the installation procedure.
   All dependencies will be downloaded from the internet.
   Do **not** use a metered, mobile connection!
-* Install `Anaconda`_, following the `installation instructions <https://docs.anaconda.com/anaconda/install/>`_ for your OS.
+* Install `Conda`_.
+  We highly recommend you use `Miniforge`_, which includes the potent `Mamba`_ package manager.
+  Download the installer suitable for your system and follow the respective installation instructions.
+  We do **not** recommend using ``conda`` anymore, see :ref:`conda-instead-of-mamba`.
```

Comment:
I find this thing confusing myself. Here's how I understand it:
* Conda: Both the `conda` package manager and the repository "ecosystem" around it.
* Anaconda: The company managing Conda. But also one of the main Conda distributions ("Anaconda Free").
* Miniconda: A boiled down distribution of Conda.
* Miniforge: A Conda distribution that is especially tailored towards the community-based "conda-forge". Comes with `conda` *and* `mamba`
* Mamba/`mamba`: A community-built replacement of the `conda` package manager (only).

In this picture, "Mamba" builds "Conda" environments, but replaces the `conda` command line tool. That's why I always talk about (capitalized) Conda in the text and use `mamba` in the commands. This becomes confusing when I mention the `conda` program. Do you have a suggestion how I can clear this up concisely?

Reply:
I see what you mean. I think it will be best if I add this info to the overview I am working on in the installation guide. To prevent misunderstandings regarding this specific chunck of text I made a suggestion to clarify that the "Conda" we are asking to install and the "conda" we recommend not using anymore are two different things. Later we can add a link to the overview explaining this stuff in more detail.

Reply:
I think this has become clear now. Thank you! If you agree, please resolve this and I will merge 🙂